openhcl/virt_mshv_vtl/src/lib.rs

```diff
@@ -2177,6 +2147,28 @@ impl UhPartitionInner {
             false
         }
     }
+
+    /// Gets the CPUID result, applying any necessary runtime modifications.
+    #[cfg(guest_arch = "x86_64")]
+    fn cpuid_result(&self, eax: u32, ecx: u32, default: &[u32; 4]) -> [u32; 4] {
+        let r = self.cpuid.result(eax, ecx, default);
```

Comment:
Can you throw this in here for me?

`// TODO TDX GUEST VSM: Consider changing TLB hypercall flag too`

We only need TLB hypercalls when using guest vsm, so when we revoke we could stop advertising that recommendation too. Not using them will likely always be more performant than using them, no matter how much optimization we apply.